3.0 updates

bicknellr opened the pull request:
- Unstable auto-formatting.
- Async demo doesn't work on Firefox: the name specifier in the dynamic import isn't being rewritten.

bicknellr committed:
update .travis.yml

bicknellr committed:
update package.json, regenerate package-lock.json

bicknellr committed:
format

bicknellr committed:
Fix comments moved by modulizer.

bicknellr committed:
Update test script references.

bicknellr committed:
Remove `index.html`, regenerate `package-lock.json`.

bicknellr committed:
Update demo script references; revert modulizer's @apply changes.

bicknellr committed:
format

aomarks commented on demo/index.html:
```diff
@@ -11,35 +11,27 @@
+    import '@polymer/iron-demo-helpers/demo-pages-shared-styles.js';
+
+    const template = document.createElement('template');
+    template.innerHTML = `
```
does it not work if you just put this HTML in the document like it originally was?

e111077 replied:
I agree with Al, I think this is the best solution for now. It looks like clang-format is shitting itself with template literals

bicknellr replied:
Same issue as [this comment I left on iron-fit-behavior](https://github.com/PolymerElements/iron-fit-behavior/pull/93#discussion_r199027618). :/ I'll ask @azakus about this tomorrow.

bicknellr committed:
Fix references in `demo/async.html`.

bicknellr committed:
Use unpkg.com in demos for loading polyfills.

bicknellr committed:
Fix async demo.

bicknellr committed:
Use the webcomponentsjs loader instead of the bundle.

bicknellr committed:
Exclude files from formatting which don't format stably. (...)

These files have template strings which the formatter indents further on each
format.

dfreedm reviewed (commented):
Travis changes LGTM

bicknellr committed:
Remove `index.html` entry from `manifest.json`.

bicknellr committed:
Add delete entry for `index.html` in `manifest.json`.

bicknellr committed:
Update webmat.

bicknellr committed:
Remove `formatconfig.json`.

bicknellr committed:
Update README.md

bicknellr committed:
Remove unused @polymer/iron-icons dependency.

bicknellr committed:
Update `iron-icon.js` comments.

bicknellr committed:
Update iron-icons demo link.

bicknellr committed:
Add iron-icon import to example.

bicknellr committed:
Remove `?dom=shadow` from tests.

bicknellr committed:
Revert "Use the webcomponentsjs loader instead of the bundle." (tests only)

This reverts commit 4dd5de3273f792e6adbceb61d3f83e9c41850c25.

bicknellr committed:
Use latest Node.js version when testing on Travis.

bicknellr committed:
format

bicknellr committed:
Remove `=""` from boolean attributes.
